src/packages/admin-ui-components/src/detail-panel/fields/markdown-field/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { EditorProvider } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import Link from '@tiptap/extension-link';
+import { useField } from 'formik';
+import { EntityField } from '../../../utils';
+import { MenuBar } from './menu-bar';
+import styles from './styles.module.css';
+
+const extensions = [
+	StarterKit.configure({
+		bulletList: {
+			keepMarks: true,
+			keepAttributes: false,
+		},
+		orderedList: {
+			keepMarks: true,
+			keepAttributes: false,
+		},
+	}),
+	Link.configure({
+		HTMLAttributes: {
+			rel: 'noopener noreferrer',
+			target: '_blank',
+		},
+		openOnClick: false,
+	}),
+];
+
+interface Props {
+	field: EntityField;
+	isReadOnly: boolean;
+	options: Record<string, unknown>;
```

Comment:
Is it possible to get options more strictly typed here? Might be possible using the react utility type ComponentProps: `ComponentProps<typeof MenuBar>['options'];